modules/axhal/src/platform/bsta1000b_fada_aarch64/utils.rs

```diff
@@ -0,0 +1,67 @@
+use core::ptr::{read_volatile, write_volatile};
+use crate::time::{busy_wait, Duration};
+use crate::mem::phys_to_virt;
+
+/// 微秒(us) 延时器
+pub fn usdelay(us: u64) {
```

Comment:
Remove this function, use `axhal::time::busy_wait` directly

Reply:
Okay